src/pages/KioskMode.tsx

```
@@ -131,8 +138,9 @@ export const KioskMode: React.FC = () => {
                   </div>
                   <h2 className="text-8xl font-black text-white mb-4 tracking-tight">WELCOME!</h2>
                   <p className="text-5xl font-bold text-green-100">{result?.userName}</p>
-                  <p className="text-3xl text-green-200 mt-4">Check-in successful</p>
-                </>
+                  <p className="text-3xl text-green-200 mt-4">
+                    {result?.isOfflineCheckIn ? "Checked In (Offline Mode)" : "Check-in successful"}
+                  </p>                </>
```

ESLint

[error] Insert `{"·"}⏎`

(prettier/prettier)